dali/test/python/operator_2/test_remap.py

```diff
@@ -160,14 +158,11 @@ def benchmark_remap_isolated(self, map_mode):
         dpipe = remap_pipe(
             "dali", maps, self.img_size, **self.common_dali_pipe_params, prefetch_queue_depth=1
         )
-        dpipe.build()
```

Comment:
This is a benchmark; please restore, we don't want to measure build.

Reply:
reverted